templates/definition/vehicle/ioBroker.bmw.yaml

```
@@ -0,0 +1,64 @@
+template: ioBroker.bmw
+products:
+  - brand: ioBroker.bmw
+requirements:
```

Sollte analog zu Teslamate als `generic` einsortiet werden. Dann landet der in der Doku unten im abgetrennten Bereich.

https://github.com/evcc-io/evcc/blob/master/templates/definition/vehicle/teslamate.yaml#L5
```suggestion
group: generic
requirements:
```